app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
+	end
+
+	def print_fib(range)
+		range.each do |n|
+			p fib(n)
+		end
+	end
+
+	private
+		def fib(n)
+			return 1 if n <= 1
+			fib(n-2) + fib(n-1)
+		end
+end
```

Comment:
__[rubocop]__
* Final newline missing. [:ref](https://github.com/bbatsov/ruby-style-guide#newline-eof)
* `end` at 17, 4 is not aligned with `def` at 14, 2.

<!-- Comment by [SideCI](https://sideci.com) -->